File: js/anagram2.js
exports.anagramsFor = function(word, listOfWords) {
    let anagrams=[]
    for (i=0;i<listOfWords.length; i++){
        counter=0
        //console.log(listOfWords[i])
        for (j=0; j<word.length;j++){
            //console.log(word[j])
            let compareWord=listOfWords[i].split(word[j]).length-1
            //console.log(compareWord)
            let compareVar=word.split(word[j]).length-1
            //console.log(compareVar)
            if(compareVar===compareWord){
                counter++
            }
        }
        if(counter===listOfWords[i].length){
            anagrams.push(listOfWords[i])
        }
    }
    return(anagrams)
};

//anagramsFor("threads", listOfWords)

File: python/anagram2.py
# Don't forget to run the tests (and create some of your own)


def anagrams_for(word, list_of_words):
    anagrams=[]
    for i in list_of_words:
        counter= 0
        for j in word:
            compare_one=i.count(j)
            compare_two=word.count(j)
            if compare_one == compare_two:
                counter += 1
        if counter==len(i):
            anagrams.append(i)
    return anagrams